packages/api/server/ws.mts

```diff
@@ -390,6 +390,13 @@ async function cellUpdate(payload: CellUpdatePayloadType) {
       `No cell exists for session '${payload.sessionId}' and cell '${payload.cellId}'`,
     );
   }
+  if (
```

Comment:
Title validation happens in `session.mts` in a function called `updateTitleCell`. In there, we use a Zod schema to handle validation. That schema currently does not cap the length but that seems like the place to add this validation rule.

Currently, the zod schema parsing will throw an exception if there's an issue. So long as the client also implements  validation, then I think it's fine for the exception to crash since it would have to be some sort of bizarre edge case or bug in client code if that were to happen.